src/lib/y2security/autoinst/lsm_config_reader.rb

```diff
@@ -0,0 +1,58 @@
+require "y2security/lsm/config"
+
+module Y2Security
+  module Autoinst
+    # This class is responsible of reading the LSM configuration declared in the AutoYaST profile
```

Comment:
Let's mention the full name at least once.
```suggestion
    # This class is responsible of reading the Linux Security Module configuration declared in the AutoYaST profile
```